up-l1/zenoh.adoc

```diff
@@ -181,19 +170,19 @@ The section provides the details how it works in different message types.
 ==== Transfer Publish message
 
 1. A uEntity (`origin-authority/origin-entity`) creates a message with
-    * source: `up://origin-authority/origin-entity/origin-version/origin-resource.instance#message`
+    * source: `up://origin-authority/origin-entity/origin-version/origin-resource`
     * sink: `-`
 
 2. `UTransport.send` maps source (**origin UUri**) to Zenoh key
-    * local (w/o origin-authority): `upl/` + `origin-resource-id + origin-entity-id + origin-version`
-    * remote (with origin-authority): `upr/` + `origin-authority-id/` + `origin-resource-id + origin-entity-id + origin-version`
+    * `up/` + `origin-authority/empty/origin-entity/origin-version/origin-resource`
```

Comment:
That would then be
`up/` + `origin-authority/{}/origin-entity/origin-version/origin-resource`
I guess

Reply:
You're right. I forgot to update here...